### ISSUE TITLE
Blocks not appearing in 1.15 map art litematica

### ISSUE THREAD

**Issue by pattywacka** [maruohon/litematica#188]
**NOTE: Please don't post "How do I do X?" type issues on this issue tracker. Such help requests should happen on Discord instead: https://discord.gg/VNVG3Wk (or if the previous link is broken: https://discord.gg/scicraft -> #masas-mods)**

**Describe the bug**
The the map art has over 50 blocks being used, and the map art litematica , when shown in the world, only has 5 blocks being used

**To Reproduce** (if not clear from the description already)

**Expected behavior** (if not clear from the description)

**Screenshots**

**Additional context**

**Comment by maruohon** [comment]
If this is from the mapartcraft site, then you should download the schematic for the same Minecraft version that you are using, so in this case for 1.15.2.

I'm currently working on proper schematic conversion support between Minecraft versions, but it will still take time before that plus the other features that I still need to do before the next updates are all done.

**Comment by pattywacka** [comment]
Yes I have been using the same versions, but sure, should i just use the more manual way of making the map-art in creative and then moving it to survival.

**Comment by maruohon** [comment]
There is no reason why the blocks should vanish if the file is generated for the same Minecraft version. I think there used to be an issue on the mapartcraft site that if you first save a schematic let's say for 1.12.2 and then you change the selected Minecraft version with the dropdown to for example 1.15.2, it wouldn't update unless you refresh the page, or something like that. (ie. it would still actually output stuff in the 1.12.2 format)
